<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerDashboard</class>
 <widget class="QMainWindow" name="ServerDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>601</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ServerDashboard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/server/theme/winicon</normaloff>:/server/theme/winicon</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="CustomTabWidget" name="dashboardCenter">
      <property name="toolTip">
       <string>Dashboard</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/server/theme/home</normaloff>:/server/theme/home</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>80</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>80</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/server/theme/winicon);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>450</width>
                <height>80</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>80</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;欢迎使用&lt;span style=&quot; font-size:18pt; font-weight:600; color:#cccccc;&quot;&gt;TeachingDashboard&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="indent">
               <number>50</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="studentInfo">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="toolTip">
               <string>学生信息</string>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>学生信息</string>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="classInfo">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="toolTip">
               <string>班级信息</string>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>班级信息</string>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="questionInfo">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="toolTip">
               <string>问题信息</string>
              </property>
              <property name="text">
               <string>问题信息</string>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>300</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="title">
               <string>学生班级信息</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border-image: url(:/server/theme/winicon);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="studentClassInfoMaintain">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="toolTip">
                    <string>学生班级信息维护</string>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string>学生班级信息维护</string>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="studentClassInfoView">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="toolTip">
                    <string>学生班级信息查看</string>
                   </property>
                   <property name="text">
                    <string>学生班级信息查看</string>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="studentClassInfoManage">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="toolTip">
                    <string>学生班级信息管理</string>
                   </property>
                   <property name="text">
                    <string>学生班级信息管理</string>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>300</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>问题信息</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border-image: url(:/server/theme/winicon);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="questionMaintain">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="toolTip">
                    <string>问题信息维护</string>
                   </property>
                   <property name="text">
                    <string>问题信息维护</string>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="questionInfoView_2">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="toolTip">
                    <string>问题信息查看</string>
                   </property>
                   <property name="text">
                    <string>问题信息查看</string>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="questionInfoAnalysis">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="toolTip">
                    <string>问题信息分析</string>
                   </property>
                   <property name="text">
                    <string>问题信息分析</string>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>300</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>课件信息</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border-image: url(:/server/theme/winicon);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="courseInfoMaintain_2">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="toolTip">
                    <string>课件信息维护</string>
                   </property>
                   <property name="text">
                    <string>课件信息维护</string>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="courseInfoView">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="toolTip">
                    <string>课件信息查看</string>
                   </property>
                   <property name="text">
                    <string>课件信息查看</string>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="courseInfoAnalysis">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="toolTip">
                    <string>课件信息分析</string>
                   </property>
                   <property name="text">
                    <string>课件信息分析</string>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_5">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>300</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>系统信息</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border-image: url(:/server/theme/winicon);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="databaseInfo">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="toolTip">
                    <string>数据库信息</string>
                   </property>
                   <property name="text">
                    <string>数据库信息</string>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="systemSetting">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="toolTip">
                    <string>系统设置</string>
                   </property>
                   <property name="text">
                    <string>系统设置</string>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="systemBroad">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="toolTip">
                    <string>信息广播信息</string>
                   </property>
                   <property name="text">
                    <string>广播信息</string>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="closeCurrentItem"/>
    <addaction name="closeAllItem"/>
    <addaction name="separator"/>
    <addaction name="systemMMAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="EditMenu">
    <property name="title">
     <string>信息编辑(&amp;E)</string>
    </property>
    <addaction name="studentEditAction"/>
    <addaction name="separator"/>
    <addaction name="classEditAction"/>
    <addaction name="separator"/>
    <addaction name="courseEditAction"/>
    <addaction name="separator"/>
    <addaction name="broardEditAction"/>
   </widget>
   <widget class="QMenu" name="DBMenu">
    <property name="title">
     <string>数据库(&amp;D)</string>
    </property>
    <addaction name="dataMMAction"/>
    <addaction name="separator"/>
    <addaction name="dataExportAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="helpContentAction"/>
    <addaction name="separator"/>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="EditMenu"/>
   <addaction name="DBMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="closeCurrentItem">
   <property name="text">
    <string>关闭当前标签页(&amp;C)</string>
   </property>
  </action>
  <action name="closeAllItem">
   <property name="text">
    <string>关闭所有标签页(&amp;A)</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>退出系统(&amp;Q)</string>
   </property>
  </action>
  <action name="studentEditAction">
   <property name="text">
    <string>学生信息(&amp;S)</string>
   </property>
  </action>
  <action name="classEditAction">
   <property name="text">
    <string>班级信息(&amp;C)</string>
   </property>
  </action>
  <action name="courseEditAction">
   <property name="text">
    <string>课件信息(&amp;K)</string>
   </property>
  </action>
  <action name="broardEditAction">
   <property name="text">
    <string>问题信息(&amp;Q)</string>
   </property>
  </action>
  <action name="dataMMAction">
   <property name="text">
    <string>数据管理(&amp;M)</string>
   </property>
  </action>
  <action name="dataExportAction">
   <property name="text">
    <string>数据导出(&amp;E)</string>
   </property>
  </action>
  <action name="startServerAction">
   <property name="text">
    <string>启动服务器</string>
   </property>
  </action>
  <action name="stopServerAction">
   <property name="text">
    <string>关闭服务器</string>
   </property>
  </action>
  <action name="systemMMAction">
   <property name="text">
    <string>系统设置(&amp;S)</string>
   </property>
  </action>
  <action name="helpContentAction">
   <property name="text">
    <string>帮助主题(&amp;T)</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomTabWidget</class>
   <extends>QTabWidget</extends>
   <header location="global">customtabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
